docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,318 @@
+# Installing With SR-IOV Compute Nodes
+
+## Table of Contents
+
+- [Pre-Install Requirements](#pre-install-requirements)
+- [Deploying an OpenShift Control Plane using IPI](#deploying-an-openshift-control-plane-using-ipi)
+- [Deploying an OpenShift Control Plane using UPI](#deploying-an-openshift-control-plane-using-upi)
+- [Preparing your OpenShift Cluster to use SR-IOV Networks](#preparing-your-openshift-cluster-to-use-sr-iov-networks)
+- [Creating SR-IOV Networks for Compute Machines](#creating-sr-iov-networks-for-compute-machines)
+- [Creating SR-IOV Compute Nodes in IPI](#creating-sr-iov-compute-nodes-in-ipi)
+- [Creating SR-IOV Compute Nodes in UPI](#creating-sr-iov-compute-nodes-in-upi)
+
+## Pre-Install Requirements
+
+Using single root I/O virtualization (SR-IOV) networking as an additional network in OpenShift can be beneficial for applications
+that require high bandwidth and low latency. If you are planning an OpenStack deployment with SR-IOV NICs enabled, refer to [the OSP 16.1 installation documentation](osp-sriov-install). Before installing, make sure that the network interface cards your OpenStack nodes are using [are supported](supported-nics) for use with SR-IOV
+in OpenShift, and that your tenant has access to them.
+
+Deploying compute nodes with SR-IOV networks is supported as a day 2, or post install, operation in the installer for both IPI and UPI workflows. Once you have verified that your OpenStack cluster can support SR-IOV in OpenShift, you can use the installation workflow of your choice to deploy an OpenShift control-plane. Only the OpenShift control-plane needs to be operational to use this feature. For clarification, SR-IOV workers can be deployed alongside regular workers without issue, but for these examples we make the assumption that the user wants only SR-IOV compute nodes. Below, we have provided some example workflows for deploying an OpenShift control-plane.
+
+## Deploying an OpenShift Control Plane using IPI
+
+To stand up only a control plane with the IPI installer, the only change that needs to be made is to scale the compute nodes to zero. This can easily be done in the install config by changing the number of replicas in the compute machine pool to zero. This minimal example will show you how to do this. Consult the [cusomization documentation](machine-pool-customizations) for more detailed information about what changes to the machinepools are available.
```

Comment:
before scaling workers (compute) to zero, you have to evacuate certain services. 
See https://issues.redhat.com/browse/OSASINFRA-1991 for details. and reference on how to do this.

Reply:
this is before Install, so you shouldn't have to worry about any of that